lms/djangoapps/staticbook/views.py

```diff
@@ -100,7 +102,15 @@ def pdf_index(request, course_id, book_index, chapter=None, page=None):
         for entry in textbook['chapters']:
             entry['url'] = remap_static_url(entry['url'], course)
         if chapter is not None:
-            current_chapter = textbook['chapters'][int(chapter) - 1]
+            if int(chapter) <= (len(textbook['chapters'])):
+                current_chapter = textbook['chapters'][int(chapter) - 1]
+            else:
+                return redirect(
+                    reverse(
+                        pdf_index,
+                        kwargs={'course_id': course_id, 'book_index': book_index, 'chapter': 0}
```

Comment:
Agreed with @awais786 , we should redirect to the chapter `1`. (it looks like chapters are 1-indexed, not 0-indexed)